src/scripts.js

```diff
@@ -13,3 +13,7 @@ console.log('This is the JavaScript entry file - your code begins here.');
 
 import userData from './data/users';
 import UserRepository from './UserRepository';
+import User from './User';
+
+console.log(userData);
+console.log(user);
```

Comment:
If you pull up the site, you'll be able to see the data on the console.

Reply:
Good thinking!